### DISCUSSION TITLE
Maven Dependencies and Plugin not found error on Intellij IDEA

### DISCUSSION THREAD

**Discussion by mdxabu** [community/community#106299] [Programming Help]
# Help me to fix this in intellij IDEA
I can't find add dependency button and tried many things too, no use, Help me to fix this!!!
![Screenshot from 2024-02-05 16-49-23](https://github.com/community/community/assets/115330277/8cc2de36-c1f1-4591-b235-15ef6db902dc)

![Screenshot from 2024-02-05 16-49-48](https://github.com/community/community/assets/115330277/731bad91-e986-4464-811f-e9654055b469)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mdxabu ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
